MachineLearningPrototype/dataCollection/collect_data.py

```diff
@@ -45,6 +45,8 @@ def store_issue_details(issue, repo, db):
 	issue_content_path = f'./data/GitHubData/IssueContent/{issue_uid}'
 	# test_output_path = f'./data/GitHubData/TestOutput/{issue_uid}'
 	issue_content_quotation_path = f'./data/GitHubData/IssueContentQuote/{issue_uid}'
+	issue_quoted_jenkins_links = f'./data/GitHubData/JenkinsOutput/{issue_uid}'
+    	issue_quoted_test_names = f'./data/GitHubData/JenkinsOutput/{issue_uid}'
```

Comment:
We do not need to store these two variables in file. For this PR, you only need to save these variables in db.